client/src/features/home/LandingPage.js

```diff
@@ -10,16 +10,48 @@ import HamburgerNav from "./HamburgerNav";
 const LandingPage = () => {
   const cards = [
     {
-      heading: "Order1",
-      description: "Descption1",
+      heading: "Join 100devs",
+      description: (
+        <>
+          <a
+            className="underline font-bold"
+            href={`https://leonnoel.com/100devs/`}
```

Comment:
nit: here and below: `{}` in react is only needed if you need JS values, if it's just a string you can pass it as a string

```suggestion
            href="https://leonnoel.com/100devs/"
```